daprdocs/content/en/developing-applications/local-development/multi-app-dapr-run/multi-app-overview.md

```diff
@@ -7,23 +7,24 @@ description: Run multiple applications with one CLI command
 ---
 
 {{% alert title="Note" color="primary" %}}
- Multi-App Run is currently a preview feature only supported in Linux/MacOS. 
+ Multi-App Run for **Kubernetes** is currently a preview feature only supported in Linux/MacOS.
```

Comment:
@mukundansundar - Can you confirm this is true? Does it work on Windows.

Reply:
Was validated by @shivamkm07 that it works on windows